bin/wmagent-resource-control

```diff
@@ -96,20 +97,22 @@ def createOptionParser():
                            help="Use SiteDB emulator for unittests")
     myOptParser.add_option("--add-disk", dest="addDisk", default=True, action="store_false",
                            help="Use it to explicitly add the Disk endpoints together with the other T1s")
+    myOptParser.add_option("--use-Cric", dest="useCric", default=False, action="store_true",
+                           help="Query CRIC Service instead of SiteDB")
 
     return myOptParser
 
 
-def getCMSSiteInfo(pattern=''):
+def getCMSSiteInfo(pattern='', cricObj=None):
```

Comment:
This is just my preference, you don't have to change the code. 😄 
Since cricObj cannot be None. function signature could be below.
```
def getCMSSiteInfo(cricObj, pattern='')
```

Reply:
I agree! I'll update it